ui/src/authors/components/AssignOwnProfileAction.jsx

```diff
@@ -81,6 +82,12 @@ AssignOwnProfileAction.propTypes = {
   disabled: PropTypes.bool,
   disabledAssignAction: PropTypes.bool,
   numberOfSelected: PropTypes.number,
+  claimingTooltip: PropTypes.string,
+};
+
+AssignOwnProfileAction.defaultProps = {
+  claimingTooltip: 'This paper is already claimed',
+  numberOfSelected: 1,
```

Comment:
why do we need `numberOfSelected` 🤔 we are not use it anywhere?

Reply:
We use it in lines 64 and 71. It's set to default 1 to work correctly with [AssignOneOwnProfileContainer](https://github.com/inspirehep/inspirehep/blob/master/ui/src/authors/containers/AssignOneOwnProfileContainer.jsx) where we only have one selected paper. And in [AssignAllOwnProfileActionContainer](https://github.com/inspirehep/inspirehep/blob/master/ui/src/authors/containers/AssignAllOwnProfileActionContainer.jsx) I pass it as props because there there can be more that one paper selected :)